backend/app/verification.py

```diff
@@ -884,6 +912,78 @@ def unverify(
         worker.republish_app.send(app_id)
 
 
+@router.post("/{app_id}/enroll", status_code=204, tags=["verification"])
```

Comment:
I'm struggling with the name here - I guess this is the endpoint for switching from a github repo to a self managed one?